engine-tests/src/main/java/org/terasology/ModuleEnvironmentTest.java

```diff
@@ -37,9 +23,9 @@
 
     @BeforeEach
     public void before() throws Exception {
-        final JavaArchive homeArchive = ShrinkWrap.create(JavaArchive.class);
-        final FileSystem vfs = ShrinkWrapFileSystems.newFileSystem(homeArchive);
-        PathManager.getInstance().useOverrideHomePath(vfs.getPath(""));
+        Path tempHome = Files.createTempDirectory("terasology-met", (FileAttribute<?>[]) null);
```

Comment:
Would [`@TempDir`](https://junit.org/junit5/docs/current/user-guide/#writing-tests-built-in-extensions-TempDirectory).work here too? I guess if the `@BeforeEach` annotation works here, `@TempDir` should too.